tests/lib/eslint/eslint.js

```diff
@@ -1697,14 +1722,14 @@ describe("ESLint", () => {
 						message:
 							"The 'jiti' library is required for loading TypeScript configuration files. Make sure to install it.",
 					});
+
+					loadJitiStub.restore();
```

Comment:
Why this change? The stubs are restored automatically by `sinon` after each test.